File: Aula 01/projeto/view/ExibicaoView.py
class ExibicaoView:

  @staticmethod
  def exibir_lista(lista: list, frase: str):
    print(frase)
    for item in lista:
      print(item)
    print("--------------------------")
    print(f"Total de registros: {len(lista)}")

  @staticmethod
  def exibir_tempo_execucao(rotina: int, tempo_ms: int):
    print(f"Tempo (ms) rotina {rotina}: {tempo_ms}")

File: Aula 02/projeto/view/ExibicaoView.cs
using System;
using System.Diagnostics;

namespace ProjetoOrdenacao.View
{
    public class ExibicaoView
    {
        public static void ExibirLista(List<int> lista)
        {
            foreach (int item in lista)
            {
                Console.WriteLine(item);
            }
        }

        public static void ExibirTempo(Stopwatch sw, string frase)
        {
            Console.WriteLine(frase + " (ms): " + sw.ElapsedMilliseconds);
        }
    }
}
